test/Microsoft.Azure.Devices.Edge.Test.Common/DaemonConfiguration.cs

```diff
@@ -209,6 +230,19 @@ public void RemoveCertificates()
             this.config[Service.Certd].Document.RemoveIfExists("preloaded_certs.iotedge-trust-bundle");
         }
 
+        public void AddPrincipal(string name, uint uid)
```

Comment:
For completeness, should we add an optional type parameter for local identity testing?

Reply:
Fixed.